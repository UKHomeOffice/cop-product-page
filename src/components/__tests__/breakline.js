import React from "react";
import renderer from "react-test-renderer";
import BreakLine from "../Break-line";

describe("breakline", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<BreakLine/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct breakline 'pixel' prop passed in", () => {
    const tree = renderer
      .create(<BreakLine pixels={"2"}/>);
    const testInstance = tree.root;
    expect(testInstance.props).toHaveProperty('pixels', '2');
  });
});