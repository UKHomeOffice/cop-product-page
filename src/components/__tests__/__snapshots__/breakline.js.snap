// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breakline renders correctly 1`] = `
<div
  style={
    Object {
      "marginTop": "undefinedpx",
    }
  }
/>
`;
